grobid-core/src/main/java/org/grobid/core/document/TEIFormatter.java

```diff
@@ -1641,6 +1645,9 @@ public StringBuilder toTEITextPiece(StringBuilder buffer,
                         if (pos > matchingPosition.start)
                             break;
 
+                        if (pos >= matchingPosition.start) 
```

Comment:
This seems duplicated.

Reply:
It comes from your change here: https://github.com/kermitt2/grobid/pull/1125

So the `>` should be kept

Reply:
fixed...